Bottom: aa604f138d83a8a0d423e7c154e619ef82c7f20a
Top:    488bdc823dc3a40b067f19cb78d6f7289bfd5a6d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 22:42:06 +0100

Refresh of hid-gtk-drop-in-gl-renderer-wo

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 42f5e45..71759ca 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -289,13 +289,6 @@ ghid_use_mask (int use_it)
    *       once during rendering (for the solder-mask layer).
    */
 
-  /* THE FOLLOWING IS COMPLETE ABUSE OF THIS MASK RENDERING API... NOT IMPLEMENTED */
-  if (use_it == HID_LIVE_DRAWING ||
-      use_it == HID_LIVE_DRAWING_OFF ||
-      use_it == HID_FLUSH_DRAW_Q) {
-    return;
-  }
-
   if (use_it == cur_mask)
     return;
